cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -838,7 +838,8 @@ SwiftGenerator::writeProxyDocSummary(IceInternal::Output& out, const InterfaceDe
 
     if (!doc.misc.empty())
     {
-        writeDocLines(out, doc.misc, false);
```

Comment:
This was arguably worse than #2088. It caused us to just slap the `@remarks` on whatever line we were already writing to. No compiler error, but a silent failure:
```
/// Process Methods:
///
///  - writeMessage: Write a message @remarks something.
```